pkg/vere/io/ames.c

```diff
@@ -2752,6 +2758,51 @@ _ames_recv_cb(uv_udp_t*        wax_u,
   }
 }
 
+static void natpmp_init(uv_timer_t* handle);
+
+static void
+natpmp_cb(uv_poll_t* handle,
+          int        status,
+          int        events)
+{
+  u3_ames* sam_u = handle->data;
+
+  natpmpresp_t response;
+  c3_w r = readnatpmpresponseorretry(&sam_u->nat_u.req_u, &response);
```

Comment:
This assignment converts a signed `int` to an unsigned `c3_w`. You should `c3_i` (and suffix the variable name with `_i`).